README.md

```diff
@@ -1,37 +1,44 @@
-# YuniKorn - A Universal Scheduler
+# Apache YuniKorn (incubating) - A Universal Scheduler
 
 ![Green Build](https://github.com/apache/incubator-yunikorn-core/workflows/Green%20Build/badge.svg)
 
 <img src="https://raw.githubusercontent.com/apache/incubator-yunikorn-core/master/docs/images/logo/yunikorn-logo-blue.png" width="200">
 
 ----
 
-YuniKorn is a light-weighted, universal resource scheduler for container orchestrator systems.
+Apache YuniKorn (incubating) is a light-weighted, universal resource scheduler for container orchestrator systems.
 It was created to achieve fine-grained resource sharing for various workloads efficiently on a large scale, multi-tenant,
 and cloud-native environment. YuniKorn brings a unified, cross-platform scheduling experience for mixed workloads consists
-of stateless batch workloads and stateful services. Support for but not limited to, YARN and Kubernetes.
+of stateless batch workloads and stateful services. 
+
+YuniKorn now supports K8s and can be deployed as a custom K8s scheduler. YuniKorn's architecture design also allows adding different
+shim layer and adopt to different ResourceManager implementation including Apache Hadoop YARN, or any other systems. 
 
 ## Architecture
 
 Following chart illustrates the high-level architecture of YuniKorn.
 
-![Architecture](docs/images/architecture.jpg)
+![Architecture](docs/images/architecture.png)
 
 YuniKorn consists of the following components spread over multiple code repositories.
 
 - _Scheduler core_: the brain of the scheduler, which makes placement decisions (Allocate container X on node Y)
   according to pre configured policies. See more in current repo [yunikorn-core](https://github.com/apache/incubator-yunikorn-core).
+  _Scheduler core_ is written in GoLang and implemented in a way to be agnostic to scheduler implementation.
```

Comment:
weird to state that just about the core, would not add it at all

Reply:
The reason I add this because many people confused and ask: is it copy of YARN scheduling code? I will add it to the top level.